packages/performance/src/services/remote_config_service.test.ts

```diff
@@ -69,7 +70,12 @@ describe('Performance Monitoring > remote_config_service', () => {
 
   beforeEach(() => {
     fetchStub = stub(self, 'fetch');
-    storageGetItemStub = stub(self.localStorage, 'getItem');
+    // we need to stub the entire localStorage, because storage can't be stubbed in Firefox and IE.
+    // stubbing on self(window) seems to only work the first time (at least in Firefox), the subsequent
+    // tests will have the same stub. stub.reset() in afterEach doesn't help either. As a result, we stub on ApiInstance
```

Comment:
Line length. Also, missing period at the end.

Reply:
done.